website/pages/docs/k8s/installation/multi-cluster/kubernetes.mdx

```diff
@@ -0,0 +1,388 @@
+    secretKey: gossipEncryptionKey
+connectInject:
+  enabled: true
+meshGateway:
+  enabled: true
+server:
+  # Here we're including the server config exported from the primary
+  # via the federation secret. This config includes the addresses of
+  # the primary datacenter's mesh gateways so Consul can begin federation.
+  extraVolumes:
+    - type: secret
+      name: consul-federation
+      items:
+        - key: serverConfigJSON
+          path: config.json
+      load: true
+```
+
+Modifications:
+
+1. Note `global.image: consul:1.8.0-beta1`. Kubernetes support for federation is currently only available
+   in Consul 1.8.0 beta.
+1. The Consul datacenter name is `dc2`. The primary datacenter's name was `dc1`.
+   The datacenter name in **each** federated cluster **must be unique**.
+1. ACLs are enabled in the above config file. They can be disabled by removing
+   the whole `acls` block:
+
+   ```yaml
+   acls:
+     manageSystemACLs: false
+     replicationToken:
+       secretName: consul-federation
+       secretKey: replicationToken
+   ```
+
+   If ACLs are enabled in one datacenter, they must be enabled in all datacenters
+   because in order to communicate with that one datacenter ACL tokens are required.
+
+1. Gossip encryption is enabled in the above config file. To disable it, don't
+   set the `gossipEncryption` key:
+
+   ```yaml
+   global:
+     # gossipEncryption:
+     # secretName: consul-federation
+     # secretKey: gossipEncryptionKey
+   ```
+
+   If gossip encryption is enabled in one datacenter, it must be enabled in all datacenters
+   because in order to communicate with that one datacenter the encryption key is required.
+
+1. The default mesh gateway configuration
+   creates a Kubernetes Load Balancer service. If you wish to customize the
+   mesh gateway, for example using a Node Port service or a custom DNS entry,
+   see the [Helm reference](/docs/k8s/helm#v-meshgateway) for that setting.
+
+With your `config.yaml` ready to go, follow our [Installation Guide](/docs/k8s/installation/overview)
+to install Consul on your secondary cluster(s).
+
+## Verifying Federation
+
+To verify that both datacenters are federated, run the
+`consul members -wan` command on one of the Consul server pods:
+
+```sh
+$ kubectl exec statefulset/consul-server -- consul members -wan
+Node                 Address              Status  Type    Build     Protocol  DC   Segment
+consul-server-0.dc1  10.32.4.216:8302     alive   server  1.7.0dev  2         dc1  <all>
+consul-server-0.dc2  192.168.2.173:8302   alive   server  1.7.0dev  2         dc2  <all>
+consul-server-1.dc1  10.32.5.161:8302     alive   server  1.7.0dev  2         dc1  <all>
+consul-server-1.dc2  192.168.88.64:8302   alive   server  1.7.0dev  2         dc2  <all>
+consul-server-2.dc1  10.32.1.175:8302     alive   server  1.7.0dev  2         dc1  <all>
+consul-server-2.dc2  192.168.35.174:8302  alive   server  1.7.0dev  2         dc2  <all>
```

Comment:
no big deal but is it worth "fixing" the `Build` column to be 1.8.0-beta?

Reply:
🍺